doc/exts/pylint_options.py

```diff
@@ -63,10 +63,21 @@ def _get_all_options(linter: PyLinter) -> OptionsDataDict:
     return all_options
 
 
+def _get_checker_title(checker: str) -> str:
+    result = _get_checker_configuration_rst_anchor(checker)
+    result += get_rst_title(f"``{checker.capitalize()}`` **Checker**", "-")
+    result += f":ref:`See the description of the checker itself <pylint.extensions.{checker}>`.\n\n"
```

Comment:
```suggestion
    result += f":ref:`See the description of the checker itself <pylint.extensions.{checker}>`.\n\n"
```
Here with a string we can't know if the checker is a pylint extension or a standard checker.